packages/react-components/src/organisms/SharingPrelimFindingsTable.tsx

```diff
@@ -131,7 +140,17 @@ const SharingPrelimFindingsTable: React.FC<SharingPrelimFindingsTableProps> = ({
                   </p>
                 </td>
                 <td>
-                  <p>{row.teamPercentShared}%</p>
+                  <p css={iconStyles}>
+                    <span css={valueStyles}>
+                      {!row.teamPercentShared || row.limitedData
```

Comment:
I believe we should remove this `!row.teamPercentShared`. Because the data can come with `teamPercentShared: 0` and  `limitedData: true` which can cause

<img width="1311" height="858" alt="Image" src="https://github.com/user-attachments/assets/1d349507-eaa6-4b54-b6ea-42664395db91" />